v_6_0_0/master_template.go

```diff
@@ -149,6 +149,7 @@ systemd:
       [Service]
       Type=oneshot
       TimeoutStartSec=0
+      # TODO: What's up with this?
       Environment="IMAGE={{.Cluster.Kubernetes.NetworkSetup.Docker.Image}}"
```

Comment:
Though this one I was wondering - what should it be? `.Images.NetworkSetupDockerImage` ?

Reply:
That seems fine as far as naming. I guess I missed this in v4.9.1. I passed the other tags in through [key.DefaultVersions in the operator](https://github.com/giantswarm/aws-operator/blob/61ba949f6da8acde9da99f379c40c85ecb8b42a4/service/controller/key/key.go#L224) so we can override them in the future. It would be nice if they had semver tags.

Reply:
It didn't exist in `v4.9.1` yet, I think. I'll use that one :+1: